src/System.IO.Packaging/tests/Tests.cs

```diff
@@ -3892,7 +3892,7 @@ public void ComparePackUriDifferentPack()
         }
 
         [Fact]
-        void CreatePackUriWithFragment()
+        public void CreatePackUriWithFragment()
```

Comment:
This is surprising.  We have analyzers that I thought should have caught this.
cc: @ViktorHofer

Reply:
analyzers don't run for test projects...

Reply:
> analyzers don't run for test projects...

xunit analyzers _only_ run for test projects.

Reply:
https://github.com/dotnet/corefx/blob/0b822c65aa586d9a62239ee5c862fede2543a443/src/Directory.Build.props#L52

Reply:
@ViktorHofer, I'm not sure what you're trying to show me with that Directory.Build.props line.  Why wouldn't xunit analyzers run on this xunit test project?

Reply:
Oh I thought you were talking about the roslyn code-rule that forces access modifiers to be specified.

Reply:
There is no xunit analyzer that specifies that test methods must be public: https://xunit.net/xunit.analyzers/rules/

Reply:
> There is no xunit analyzer that specifies that test methods must be public

Ugh.  That's unfortunate.  Thanks.

Reply:
>  I'm not sure what you're trying to show me with that Directory.Build.props line. Why wouldn't xunit analyzers run on this xunit test project?

I was talking about the analyzers specified here: https://github.com/dotnet/corefx/blob/master/eng/Analyzers.props. Of course xunit analyzers run on test projects but there is no rule that forces a method to be public. The roslyn analyzers would have caught that.

Reply:
> The roslyn analyzers would have caught that.

Sort of.  They'd have flagged it, but then an auto-fix would have explicitly added "private".